Documentation/Getting-Started/storage-architecture.md

```diff
@@ -21,12 +21,65 @@ specified in the Ceph custom resources (CRs) and apply the changes.
 
 Rook automatically configures the Ceph-CSI driver to mount the storage to your pods.
 
-![Rook Components on Kubernetes](ceph-storage/kubernetes.png)
-
+Rook automatically configures the Ceph-CSI driver to mount the storage to your pods.
```

Comment:
Need to remove this line, it is still duplicated from line 22
```suggestion
```

Reply:
@travisn Updated, PTAL